lib/SSpkS/Package/Package.php

```diff
@@ -201,13 +201,17 @@ public function extractIfMissing($inPkgName, $targetFile)
         }
         // Try to extract file
         $tmp_dir = sys_get_temp_dir();
-        $free_tmp = disk_free_space($tmp_dir);
-        if ($free_tmp < 2048) {
-            throw new \Exception('TMP folder only has ' . $free_tmp . ' Bytes space available. Disk full!');
+        $free_tmp = @disk_free_space($tmp_dir);
+        if (!empty($free_tmp)) {
+            if ($free_tmp < 2048) {
```

Comment:
Is it possible to refactor these two if statements into something like that ?

```php
if (!empty($free_tmp) && $free_tmp < 2048)) {
```

Reply:
Yeah sure. Updated accordingly.